api/base/exceptions.py

```diff
@@ -91,6 +91,13 @@ def json_api_exception_handler(exc, context):
     return response
 
 
+def format_validation_error(e):
+    error_list = []
+    for key, value in e.message_dict.items():
+        error_list.append('There was an issue with the {} field. {}'.format(key, value[0]))
+    return error_list
```

Comment:
👍 more readable
![screen shot 2019-01-18 at 2 05 58 pm](https://user-images.githubusercontent.com/9755598/51410833-5d0a2380-1b2b-11e9-90e5-dc8d1bb28ecd.png)
instead of
![screen shot 2019-01-18 at 2 06 06 pm](https://user-images.githubusercontent.com/9755598/51410836-609daa80-1b2b-11e9-9b33-3db8ab0aadd3.png)